src/main/java/com/scottlogic/librarygradproject/Services/BookService.java

```diff
@@ -75,12 +76,14 @@ public void put(Book book) {
 
     public void deleteAll() {
         bookRepo.deleteAll();
+        bookRepo.clearDeletedBookReservations();
```

Comment:
Couldn't you just delete all the reservations here? Also, do we really need an endpoint for deleting all books? I guess if the library burnt down...